buildpacks/jvm-function-invoker/Cargo.lock

```diff
@@ -65,6 +65,37 @@ version = "3.9.1"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "a4a45a46ab1f2412e53d3a0ade76ffad2025804294569aae387231a0cd6e0899"
 
+[[package]]
```

Comment:
(Side note) Upgrading here adds 7 new deps, due to the addition of `cargo_metadata` to `libcnb-proc-macro` for the `exec.d` feature. These won't impact the binary size, since these are proc-macro deps, and buildpacks that use `libcnb-test` (which this buildpack will start to use soon) will have to compile these anyway. Plus exec.d usage will hopefully be reasonably common (over profile.d at least), that it doesn't seem worth adding an `exec-d` feature to libcnb - however (a) mentioning here for completeness, (b) I guess it's worth us adding "check if this should be a behind a feature flag" to our mental review checklist when we add other new (and less commonly used) features/deps to libcnb in the future.